source/GlassView.Core/ICreate.cs

```diff
@@ -0,0 +1,7 @@
+namespace GlassView.Core;
+
+public interface ICreate<out TSelf, in TParameter>
```

Comment:
I haven't seen any usages for this. Is this required?

Reply:
Quite right. I'll likely delete it :-)

Reply:
Deleted.